website/versioned_docs/version-1.3.0/build-tools/sbt.md

```diff
@@ -0,0 +1,175 @@
+---
+id: version-1.3.0-sbt
+title: sbt
+sidebar_label: sbt
+original_id: sbt
+---
+
+## Export the build
+
+Install and learn how to export the build in the Installation Guide:
+
+<a class="button installationButton" href="/bloop/setup">Follow the Installation Guide</a>
+
+</br>
+
+The following page only walks you through all the configuration options that the `sbt-bloop` plugin
+allows when it's configured in your build. To learn how to install the plugin or the basic concepts
+of exporting your build via bloopp head to the installation guide linked above.
```

Comment:
bloopp -> bloop

Reply:
I see it's in the versioned_docs - not sure if it's important to change this here.

Reply:
Fixed also in the other document!